corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -113,6 +115,7 @@ hqDefine("geospatial/js/geospatial_map", [
 
         self.runCaseDisbursementAlgorithm = function (cases, users) {
             self.setBusy(true);
+            self.showParams(true);
```

Comment:
Should we hide the banner again at any point (e.g. when the error banner pops up)? My thinking is so that we make sure to not clutter the UI too much with lots of different banners.

Reply:
Good point

Reply:
Actually, now that I think of it, when the error suggests the user make adjustments to the settings I'm thinking it would be helpful to have the values in front of you.

Reply:
The user would also be able to see the values on the settings page when they go to make adjustments. But no strong feelings on showing it alongside the error as well. I'll leave the decision on keeping it up to you.